geode-core/src/main/java/org/apache/geode/internal/cache/control/InternalResourceManager.java

```diff
@@ -228,41 +224,29 @@ public void removeResourceListener(final ResourceType resourceType,
   public void deliverEventFromRemote(final ResourceEvent event) {
     assert !event.isLocal();
 
-    if (this.cache.getLogger().fineEnabled()) {
-      this.cache.getLogger()
+    if (cache.getLogger().fineEnabled()) {
```

Comment:
I would go ahead and replace all usages of the old proprietary LogWriter API (cache.getLogger() returns LogWriter) with Log4J2 Logger.

Reply:
Will do!